book/visual-effects.md

```diff
@@ -1773,7 +1772,7 @@ Note that I've specified an opacity of `1.0` for the clip `Blend`.
 
 ::: {.further}
 Implementing high-quality raster libraries is very interesting in its own
-right---check out [Real-Time Rendering][rtr-book] for more.[^cgpp]
+right---check out [*Real-Time Rendering*][rtr-book] for more.[^cgpp]
```

Comment:
I also think instead of linking to "links" at the end of the chapter, in the book these should refer the reader to the bibliography. I have a comment to that effect in the doc.